src/main/resources/templates/email/observation/plotReplaced/body.ftlh.mjml

```diff
@@ -8,19 +8,22 @@
 
         <mj-section padding="0px">
             <mj-column mj-class="body-wrapper">
-                <#if !hasPrimaryContact>
+                <mj-text mj-class="text-headline06-bold">
+                ${strings("notification.observation.monitoringPlotReplaced.email.body.1", organizationName)}
+                </mj-text>
+                <!-- htmlmin:ignore --><#if !hasPrimaryContact><!-- htmlmin:ignore -->
```

Comment:
Couldn't get this to work, taking a look in the debugger. Since we will be removing this content maybe it's not so urgent.